struct StructMap {
    1: required string long_name_foo_bar,
    2: required string long_key_bar_name,
    3: required string long_foo_key_name,
    4: required string key_foo_bar_name,
    5: required string name_bar_long_foo,
    6: required string bar_foo_key_long,
    7: required string bar_foo_long_name,
    8: required string long_foo_name_bar,
    9: required string key_foo_name_long,
    10: required string name_long_foo_key,
    11: required string foo_bar_long_name,
    12: required string key_bar_name_long,
    13: required string bar_name_long_foo,
    14: required string key_name_bar_long,
    15: required string foo_name_bar_key,
    16: required string long_key_foo_name,
    17: required string name_foo_long_key,
    18: required string key_name_bar_foo,
    19: required string bar_long_name_key,
    20: required string long_bar_foo_key,
    21: required string bar_long_key_foo,
    22: required string foo_bar_long_key,
    23: required string name_foo_key_bar,
    24: required string name_key_long_bar,
    25: required string name_key_bar_foo,
    26: required string key_bar_long_name,
    27: required string long_key_name_bar,
    28: required string key_name_long_foo,
    29: required string foo_name_key_bar,
    30: required string key_name_long_bar,
    31: required string foo_name_bar_long,
    32: required string name_long_key_foo,
    33: required string name_long_key_bar,
    34: required string name_bar_foo_long,
    35: required string long_bar_name_foo,
    36: required string bar_key_foo_name,
    37: required string name_foo_key_long,
    38: required string foo_long_key_bar,
    39: required string key_foo_name_bar,
    40: required string key_name_foo_long,
    41: required string long_foo_bar_name,
    42: required string foo_name_key_long,
    43: required string bar_key_long_foo,
    44: required string bar_foo_long_key,
    45: required string key_long_foo_name,
    46: required string name_foo_long_bar,
    47: required string foo_key_name_long,
    48: required string long_name_bar_foo,
    49: required string foo_bar_name_long,
    50: required string foo_long_key_name,
    51: required string key_bar_foo_name,
    52: required string foo_bar_key_name,
    53: required string bar_name_foo_long,
    54: required string name_bar_key_long,
    55: required string key_bar_foo_long,
    56: required string long_key_bar_foo,
    57: required string key_foo_long_bar,
    58: required string bar_foo_name_long,
    59: required string key_long_bar_foo,
    60: required string long_bar_foo_name,
    61: required string foo_long_name_bar,
    62: required string long_bar_key_name,
    63: required string long_name_foo_key,
    64: required string long_foo_name_key,
    65: required string name_long_bar_key,
    66: required string long_key_foo_bar,
    67: required string name_key_foo_long,
    68: required string name_key_long_foo,
    69: required string key_name_foo_bar,
    70: required string long_foo_key_bar,
    71: required string foo_name_long_bar,
    72: required string name_long_bar_foo,
    73: required string long_name_bar_key,
    74: required string bar_long_foo_name,
    75: required string key_long_bar_name,
    76: required string name_key_bar_long,
    77: required string foo_long_name_key,
    78: required string bar_long_foo_key,
    79: required string bar_key_name_long,
    80: required string bar_foo_key_name,
    81: required string key_long_name_bar,
    82: required string long_bar_key_foo,
    83: required string foo_name_long_key,
    84: required string key_bar_long_foo,
    85: required string long_bar_name_key,
    86: required string name_bar_long_key,
    87: required string key_foo_long_name,
    88: required string foo_key_bar_long,
    89: required string foo_bar_key_long,
    90: required string bar_long_key_name,
    91: required string name_bar_key_foo,
    92: required string bar_name_key_long,
    93: required string bar_name_long_key,
    94: required string foo_key_name_bar,
    95: required string name_bar_foo_key,
    96: required string long_name_key_foo,
    97: required string long_name_key_bar,
    98: required string foo_key_long_name,
    99: required string foo_key_long_bar,
    100: required string foo_long_bar_name,
    101: required string key_long_name_foo,
    102: required string bar_key_long_name,
    103: required string foo_bar_name_key,
    104: required string key_bar_name_foo,
    105: required string foo_long_bar_key,
    106: required string name_key_foo_bar,
    107: required string bar_name_key_foo,
    108: required string long_key_name_foo,
    109: required string name_foo_bar_long,
    110: required string bar_long_name_foo,
    111: required string foo_key_bar_name,
    112: required string name_foo_bar_key,
    113: required string bar_key_name_foo,
    114: required string name_long_foo_bar,
    115: required string long_foo_bar_key,
    116: required string key_foo_bar_long,
    117: required string bar_name_foo_key,
    118: required string bar_foo_name_key,
    119: required string key_long_foo_bar,
    120: required string bar_key_foo_long,
}
